app/views/bus_stops/index.haml

```diff
@@ -10,7 +10,11 @@ or
   = text_field_tag :name, '', required: true, class: 'name-autocomplete'
   = submit_tag 'Search'
 or
-= form_tag by_route_bus_stops_path, method: :get do
+= form_tag by_status_bus_stops_path, method: :get do
   = label_tag :number, 'Select a route'
   = select_tag :number, options_for_select(Route.pluck :number)
-  = submit_tag 'Search'
+  = submit_tag 'View stops by status'
+= form_tag by_sequence_bus_stops_path, method: :get do
+  = label_tag :number, 'Select a route'
+  = select_tag :number, options_for_select(Route.pluck :number)
```

Comment:
Omitting inner parentheses is generally something we've avoided as a point of style. But since `Route.pluck(:number)` is a piece of data, better to fetch it in the controller and sidestep the style issue altogether.